### PR TITLE
[TASK] Add plugin.[pluginName].view.defaultPid

### PR DESCRIPTION
**Pull request by brotkrueml**
See: https://github.com/TYPO3/typo3/blob/main/typo3/sysext/extbase/Classes/Service/ExtensionService.php#L148-L160
Related: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/2665
Releases: main, 11.5

### CODE REVIEW

File: Documentation/TopLevelObjects/Plugin.rst [hunk 1]
```diff
@@ -392,6 +392,27 @@ view.pluginNamespace
...
+.. container:: table-row
+
+   Property
+      view.defaultPid
+
+   Data type
+      "auto" | :t3-data-type:`integer`
+
+   Description
+      This can be used to specify a default target page ID. If this value is
```
Comment (linawolf) [inline Documentation/TopLevelObjects/Plugin.rst:409]:
```suggestion
      This setting can be used to specify a default target page ID. If this value is
```

File: Documentation/TopLevelObjects/Plugin.rst [hunk 2]
```diff
@@ -392,6 +392,27 @@ view.pluginNamespace
...
+      set, this value will be used as target page ID. If :typoscript:`defaultPid`
```
Comment (linawolf) [inline Documentation/TopLevelObjects/Plugin.rst:410]: this value, this value etc. Sounds strange

File: Documentation/TopLevelObjects/Plugin.rst [hunk 3]
```diff
@@ -392,6 +392,27 @@ view.pluginNamespace
...
+      is set to :typoscript:`auto`, a pid is determined by loading the
```
Comment (linawolf) [inline Documentation/TopLevelObjects/Plugin.rst:411]:
```suggestion
      is set to :typoscript:`auto`, a PID is determined by loading the
```

File: Documentation/TopLevelObjects/Plugin.rst [hunk 4]
```diff
@@ -392,6 +392,27 @@ view.pluginNamespace
...
+      :sql:`tt_content` record that contains this plugin. An error will be
+      thrown if more than one record matches the :sql:`list_type`.
```
Comment (linawolf) [inline Documentation/TopLevelObjects/Plugin.rst:413]: I don't get why we talk about the listtype here suddenly?

**Comment by linawolf** [comment]
@brotkrueml are you still working on this one?

**Comment by DavidBruchmann** [comment]
can we get this finished today?

**Comment by linawolf** [comment]
@brotkrueml are you still working on this PR?

**Pull request closed without merging**